packages/gamut/src/BetaSticker/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+export const BetaSticker: React.FC = () => {
+  return (
+    <svg
+      width="54"
+      height="28"
+      viewBox="0 0 54 28"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
```

Comment:
We may want to add `className` as a prop to be forwarded so that `styled(BetaSticker)()` works :) and disallow trying to call <BetaSticker>  with children :)

```suggestion
type BetaStickerProps = {
 className?: string;
 children?: never
}

export const BetaSticker: React.FC<BetaStickerProps> = (props) => {
  return (
    <svg
      width="54"
      height="28"
      viewBox="0 0 54 28"
      fill="none"
      xmlns="http://www.w3.org/2000/svg"
      {...props}
    >
```